ext/graphql_libgraphqlparser_ext/extconf.rb

```diff
@@ -1,6 +1,11 @@
 require 'mkmf'
 
-dir_config('graphql', '/usr/local/include/graphqlparser', '/usr/local/lib')
-abort 'missing libgraphqlparser' unless have_library 'graphqlparser'
+prefixes = %w(/usr /usr/local)
+prefix = prefixes.find{ |prefix| Dir["#{prefix}/lib/libgraphqlparser*"].first }
+abort 'missing libgraphqlparser' unless prefix
```

Comment:
Will this prevent people using `--with-graphql-include`? (as described in https://github.com/goco-inc/heroku-buildpack-libgraphqlparser)

Maybe it could fall back to the first item, and abort later on?